ios-contact-manager/ios-contact-manager/Controller/ContactTableViewCell.swift

```diff
@@ -1,7 +1,9 @@
 import UIKit
 
-final class ContactTableViewCell: UITableViewCell {
-
+final class ContactTableViewCell: UITableViewCell, ContactCellConfigurable {
+    @IBOutlet weak var title: UILabel!
+    @IBOutlet weak var subTitle: UILabel!
```

Comment:
```suggestion
    @IBOutlet private weak var titleLabel: UILabel!
    @IBOutlet private weak var subTitleLabel: UILabel!
```

* title, subTitle 이라고만 선언하니 모델 값처럼 느껴지네요! 네이밍 끝에 Label 을 붙여 뷰 객체임이 확실히 보이도록 이름 지어보는건 어떨까요?
* 현재 internal 접근제한자로 둬서 외부에서 참조가 가능하네요. 객체는 상태를 감추고 동작(`메서드`)으로 다른 객체와 소통하는게 좋습니다. 감추기 위해서 private 접근제한자를 쓰는 건 어떨까요?
* [옵션] 디스코드에도 말씀드린 것처럼 코드로 커스텀 셀을 만들어보세요. 
* [옵션] UITableViewCell 이나 UICollectionViewCell 은 contentView 라는게 있고 여기에다 서브뷰를 addSubview 해줘야 합니다. 
  * 왜 Cell에다 addSubview 를 하면 안되고 contentView 에다 addSubview 를 해야 하는지 고민 한번 해보세요 ㅎㅎ. 또 그냥 Cell에다 addSubview 를 하면 어떤 문제가 발생하는지도 알면 더 좋고요. 이 질문은 옵션이니 여유되실때 답변 주시면 될 것 같아요 👍

```swift
class SomeTableView: UITableView {
   init() {
       self.addSubview(UIView()) /// (X)
       self.contentView.addSubview(UIView()) /// (O)
   }
}
```